test/services/referral_customer.test.js

```diff
@@ -87,7 +87,7 @@ describe('ReferralCustomer Service', function () {
       creditCardDetails.cvc,
     );
 
-    expect(paymentMethod.id).to.match(/^card_/);
+    expect(paymentMethod.public_id).to.match(/^card_/);
```

Comment:
question: did this really change? Why in the world is our API making breaking changes?

Reply:
Not exactly a breaking change to the end user, but it is a breaking change on the test assertion. The previous `id` has been replaced with `public_id` and `id` now consists of random numbers. See below cassettes

Previous: https://github.com/EasyPost/easypost-node/blob/301150ac3da50dcc57902dd21adca70e30e1ddc6/test/cassettes/ReferralCustomer-Service_3380152635/add-a-referral-user-credit-card_1978527022/recording.har#L333

Current:  https://github.com/EasyPost/easypost-node/blob/54e03f83883d5f97622ddfe86345734abe838bd2/test/cassettes/ReferralCustomer-Service_3380152635/add-a-referral-user-credit-card_1978527022/recording.har#L337